ground/src/main/java/com/google/android/ground/ui/home/HomeScreenFragment.java

```diff
@@ -409,8 +409,15 @@ private void showSurveySelector() {
   }
 
   private void showDataCollection() {
+    String dummySurveyId = "123";
```

Comment:
Maybe add a TODO referencing the GitHub issue mentioning these are temporary.

Reply:
Good idea. Done!